client/login/magic-login/request-login-email-form.jsx

```diff
@@ -75,6 +75,9 @@ class RequestLoginEmailForm extends React.Component {
 						onDismissClick={ this.onNoticeDismiss }
 						status="is-error" />
 				}
+				<h1 className="magic-login__form-header">
+					{ translate( 'Email me a login link.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Email me a login link' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).